credit-risk/requirements.txt

```diff
@@ -1,11 +1,12 @@
-numpy==1.19.1
-pandas==1.1.0
+numpy==1.22.2
+pandas==1.4.0
 plotly==4.10.0
-pygments==2.7.1
+pygments==2.15.0
 h2o-wave<1.0
```

Comment:
https://pypi.org/project/h2o-wave/1.0.2/

Reply:
Wave 1.0 introduces a breaking change for handle_on functionality. So Wave update needs to be handled separately.